composer.json

```diff
@@ -14,10 +14,14 @@
         "php": ">=5.5"
     },
     "require-dev": {
-        "phpunit/phpunit": ">4.8",
-        "phpdocumentor/phpdocumentor": ">=2.9",
+        "jakoch/phantomjs-installer": "^2.1",
+        "phpunit/phpunit": "=4.8",
```

Comment:
Note: may need to downgrade your PHP to `5.5` before running `composer update` to build a `composer.lock` compatible with PHP 5.5, for tests to pass on 5.5.

If you need me to do that, just lemme know.